### PR TITLE
Update developer guide 

### PR DESCRIPTION
**Pull request by TerenceWongsExtravaganza**
This pull request aims to update the developer guide. Currently, the following is added:

1. Use cases for various scheme commands

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
           Use case ends.
 
+**Use case: Check eligibility for assistance**
```
Comment (nws321) [inline docs/DeveloperGuide.md]: For the use cases I updated in my PR I added till `UC05` so might want to put `UC06` onwards for ur use cases to standardise

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**MSS:**
+
+1. User chooses which person to check eligibility for assistance
```
Comment (nws321) [inline docs/DeveloperGuide.md]: Do add fullstops for all the points as thats the format used in the notes.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. SocialBook displays what schemes the person is eligible for
+
+    Use case ends.
+
+**Extensions:**
+
+* 1a. Given index is invalid.
+    * 1a1. SocialBook informs user that the index is invalid
+      Use case resumes at step 1.
```
Comment (nws321) [inline docs/DeveloperGuide.md]: Can change `at` to `from` as thats the format used in the notes too. Should resume from step 2 after changes above made.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**MSS:**
+
+1. User chooses which person to check eligibility for assistance
+2. SocialBook displays what schemes the person is eligible for
+
+    Use case ends.
+
+**Extensions:**
+
+* 1a. Given index is invalid.
```
Comment (nws321) [inline docs/DeveloperGuide.md]: Can change the phrasing to `SocialBook detects missing or invalid input.`

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * 1a1. SocialBook informs user that the index is invalid
```
Comment (nws321) [inline docs/DeveloperGuide.md]: I think can add the lines below to standardise with the way other use cases resume 
`1a2. User corrects the input and enters the command again.`
`Steps 1a1-1a2 are repeated until the user enters a correct input.`

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+      Use case resumes at step 1.
+
+
+**Use case: Add a scheme to a person**
+
+**MSS**
+1. User checks what schemes a person is eligible for.
```
Comment (nws321) [inline docs/DeveloperGuide.md]: Think can remove this point as I think MSS would assume the user already looked at eligible schemes and knows which one they r trying to add to the person

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. User selects a scheme to add to the person.
+3. SocialBook adds the scheme to the person and displays the updated information.
+
+    Use case ends.
+
+**Extensions:**
+* 2a. The given index is invalid.
+    * 2a1. SocialBook informs user that the index is invalid
```
Comment (nws321) [inline docs/DeveloperGuide.md]: same as above comment about indicating user will repeat till right input

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Use case: View what schemes a person is under**
+1. User chooses which person to check 
+2. SocialBook displays what schemes the current person is under
+
+    Use case ends.
+
+**Extensions:**
+
+* 1a. Given index is invalid.
+    * 1a1. SocialBook informs user that the index is invalid
```
Comment (nws321) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+      Use case resumes at step 1.
+
+**Use case: Delete schemes from a person**
+
+**MSS**
+1. User checks what schemes a person is under.
```
Comment (nws321) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. User selects a scheme to delete from the person.
+3. SocialBook deletes the scheme from the person and displays the updated information.
+
+   Use case ends.
+
+**Extensions:**
+* 2a. The given index is invalid.
+    * 2a1. SocialBook informs user that the index is invalid
```
Comment (nws321) [inline docs/DeveloperGuide.md]: same as above

**Review by nws321 — APPROVED** [review]
LGTM, just some standardisation with the other use cases to handle.

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (TerenceWongsExtravaganza) [reply docs/DeveloperGuide.md]: im using this to reference the mentioned use case above

**Pull request merged**